packages/router/src/splash-page.tsx

```diff
@@ -1,134 +1,538 @@
+                      <h2 className="resource-title">Tutorial</h2>
+                    </div>
+
+                    <p className="resource-description">
+                      Start here to learn how to build full-stack apps with
+                      Redwood.
+                    </p>
+                  </div>
+
+                  <div className="resource">
+                    <a
+                      className="resource-link"
+                      href="https://redwoodjs.com/docs/introduction"
+                      target="_blank"
+                      rel="noreferrer"
+                    ></a>
+                    <div className="icon-container">
+                      <svg
+                        xmlns="http://www.w3.org/2000/svg"
+                        viewBox="0 0 24 24"
+                        className="icon"
+                      >
+                        <path d="M0 0h24v24H0V0z" fill="none"></path>
+                        <path
+                          d="M13 13l-3-2.25L7 13V4H6v16h12V4h-5z"
+                          opacity=".3"
+                        ></path>
+                        <path d="M18 2H6c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zM9 4h2v5l-1-.75L9 9V4zm9 16H6V4h1v9l3-2.25L13 13V4h5v16z"></path>
+                      </svg>
+                      <h2 className="resource-title">Docs and Cookbook</h2>
+                    </div>
+
+                    <p className="resource-description">
+                      Find in-depth information about Redwood features and API.
+                    </p>
+                  </div>
+
+                  <div className="resource">
+                    <a
+                      className="resource-link"
+                      href="https://community.redwoodjs.com/"
+                      target="_blank"
+                      rel="noreferrer"
+                    ></a>
+                    <div className="icon-container">
+                      <svg
+                        xmlns="http://www.w3.org/2000/svg"
+                        viewBox="0 0 24 24"
+                        className="icon"
+                      >
+                        <path d="M0 0h24v24H0V0z" fill="none"></path>
+                        <path d="M20 2H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h14l4 4V4c0-1.1-.9-2-2-2zm0 15.17L18.83 16H4V4h16v13.17z"></path>
+                        <path d="M4 4v12h14.83L20 17.17V4z" opacity=".3"></path>
+                      </svg>
+                      <h2 className="resource-title">Join the Community</h2>
+                    </div>
+
+                    <p className="resource-description">
+                      Get help, share tips, and collaborate together on Redwood.
+                    </p>
+                  </div>
+
+                  <div className="resource">
+                    <a
+                      className="resource-link"
+                      href="https://github.com/redwoodjs/redwood/blob/main/CONTRIBUTING.md"
```

Comment:
```suggestion
                      href="https://redwoodjs.com/docs/contributing"
```

This is a better link for Contributing.